src/app/services/actions/query-action-creator-util.ts

```diff
@@ -103,9 +103,13 @@ const makeRequest = (httpVerb: string, scopes: string[]): Function => {
 
     if (query.sampleHeaders) {
       query.sampleHeaders.forEach(header => {
-        if (header.name !== '' && header.value !== '') {
-          sampleHeaders[header.name] = header.value;
+        if (header.name.toLowerCase() === 'content-type') {
```

Comment:
We should check that content-type value is set.

Reply:
Relying on the graph client to set the content type. Please add comment that content-type header is being set by Graph JavaScript client.